### PR TITLE
[Bridge 26/n] emit TokenBridgeEvent event and stitch things e2e for sui -> eth

### PR DESCRIPTION
**Pull request by longbowlu**
## Description 

as title

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and do not break anything, you can skip the following section. Otherwise, please briefly describe what has changed under the Release Notes section.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: examples/move/bridge/sources/bridge.move [hunk 1]
```diff
@@ -178,7 +187,16 @@ module bridge::bridge {
...
+        emit(TokenBridgeEvent {
+            message_type: message_types::token(),
+            seq_num: bridge_seq_num,
+            source_chain: inner.chain_id,
+            sender_address: address::to_bytes(tx_context::sender(ctx)),
+            target_chain,
+            target_address,
+            token_type: token_id,
+            amount: token_amount,
+        });
```
Comment (longbowlu) [inline examples/move/bridge/sources/bridge.move:199]: @patrickkuo

File: crates/sui-bridge/src/events.rs [hunk 2]
```diff
@@ -24,6 +28,19 @@ use sui_types::digests::TransactionDigest;
 
 // This is the event structure defined and emitted in Move
 #[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Clone)]
+pub struct MoveTokenBridgeEvent {
```
Comment (patrickkuo) [inline crates/sui-bridge/src/events.rs:31]: Do we plan to move these to SDK?

**Review by patrickkuo — APPROVED** [review]
LGTM

File: crates/sui-bridge/src/events.rs (hunk 2 above)
Comment (longbowlu) [reply crates/sui-bridge/src/events.rs:31]: I think so,yeah

**Pull request merged**